tests/scripts/installandtest.sh

```diff
@@ -26,7 +27,11 @@ if [ -z "${SKIP_INSTALL}" ]; then
     oc project ${ODHPROJECT} # in case a new project is not created
     $HOME/peak/install.sh
     echo "Sleeping for 5 min to let the KfDef install settle"
```

Comment:
Minor thing: you should move this into the `if else` because the duration of the sleep depends on `${LOCAL}`